integration/src/test/java/edu/unc/lib/boxc/integration/web/access/SearchEndpointIT.java

```diff
@@ -156,4 +156,46 @@ public void testSearchTypeParamWithFileSpecified() throws Exception {
             assertValuePresent(metadata, 1, "type", "File");
         }
     }
+
+    @Test
+    public void testSearchWithNormalSorting() throws Exception {
+        createDefaultObjects();
+        collectionFactory.createCollection(adminUnit1,
+                Map.of("title", "A first collection", "readGroup", "everyone"));
+
+        // rollup=false disables the FileObjects getting grouped into their associated Works
+        var getMethod = new HttpGet(SEARCH_URL + "/?sort=title,normal");
+
+        try (var resp = httpClient.execute(getMethod)) {
+            var metadata = getMetadataFromResponse(resp);
+
+            assertSuccessfulResponse(resp);
+            // make sure all items return
+            assertEquals(6, metadata.size());
+            // check for the collection, should be first
+            assertValuePresent(metadata, 0, "type", "Collection");
+            assertValuePresent(metadata, 0, "title", "A first collection");
```

Comment:
Assuming the rest of the objects have a defined order, it would be nice to include them here so that we demonstrate its not sorting by something else that happens to also make this go first (like date created or something). Just checking the title would be sufficient. I think it'd be fine to edit the default objects titles if that was necessary to produce a reliable sort order.

Reply:
There is a way, but it'd be way easier to edit them in the parent test class so its like that for all tests.

Reply:
done!